x-pack/platform/plugins/shared/actions/common/validate_email_addresses.ts

```diff
@@ -14,8 +14,20 @@ import { hasMustacheTemplate } from './mustache_template';
 export interface ValidateEmailAddressesOptions {
   /** treat any address which contains a mustache template as valid */
   treatMustacheTemplatesAsValid?: boolean;
+  // addresses with this option won't be validated against the allowed recipient list
+  isSender?: boolean;
 }
 
+export const isAddressMatchingSomePattern = (address: string, patterns: string[]): boolean => {
```

Comment:
first I tried using `minimatch` library but the bundle size exploded

Reply:
I'd prefer something like `minimatch` rather than using regexp's (which always get us in trouble).  Wondering if we can check with Kibana core to see if they have an alternative.

Reply:
For the record: We checked with core, there is currently no alternative. We found another library that isn't that expensive (wildcard-match), but it's not well-known and using the suggested [`escapeRegExp()`](https://lodash.com/docs/4.17.15#escapeRegExp) function covers our current needs